astroid/protocols.py

```diff
@@ -690,6 +690,9 @@ def _determine_starred_iteration_lookups(starred, target, lookups):
     if isinstance(stmt, nodes.Assign):
         value = stmt.value
         lhs = stmt.targets[0]
+        if not hasattr(lhs, "elts"):
```

Comment:
I'd prefer using an `isinstance` check here as I think it is more explicit, but I'm not sure how easy it is to come up with a list of nodes that have `elts` defined.